src/systemctl/systemctl.c

```diff
@@ -8505,6 +8512,21 @@ static int systemctl_parse_argv(int argc, char *argv[]) {
                         arg_reboot_argument = optarg;
                         break;
 
+                case ARG_TIMESTAMP_STYLE:
+                        if (!STR_IN_SET(optarg, "us", "µs", "utc", "help"))
+                                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                                       "Invalid value: %s.", optarg);
+                        if (streq(optarg, "help")) {
+                                puts("us\n"
+                                     "µs\n"
+                                     "utc");
+                                return 0;
+                        } else if (STR_IN_SET(optarg, "us", "µs"))
+                                arg_timestamp_style = TIMESTAMP_US;
+                        else if (streq(optarg, "utc"))
+                                arg_timestamp_style = TIMESTAMP_UTC;
+                        break;
```

Comment:
i still would like to see a timestamp_style_from_string() function. The thing is that we likely will add something like this everywhere where we output timestamps, sooner or later, hence we should have the helper right from the beginning...

Reply:
btw, if you also add timestamp_style_to_string() in similar fashion, then the help text can be done with DUMP_STRING_TABLE()...

Reply:
ok, I've used string-table.h and added new enums for the unicode variants - seems to be the cleanest solution, ptal